P/PhyML/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+using BinaryBuilder, Pkg
+
+name = "PhyML"
+version = v"3.3.20210609" # latest passing build
+hash = "eb1009ebef100d34696db95301ba7cb55dceeb40"
+sources = [
+    GitSource("https://github.com/stephaneguindon/phyml.git", hash),
+    DirectorySource("./bundled")
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/phyml
+install_license ${WORKSPACE}/srcdir/phyml/COPYING
+
+# disable -march=native flag
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done    
+
+# generate config scripts
+./autogen.sh
+
+# make single-process version 
+./configure ${extraflags} --enable-phyml --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make clean & make -j${nproc}
+install -Dvm 755 ./src/phyml${exeext} ${bindir}/phyml${exeext}
+
+# Make MPI version
+./configure ${extraflags} --enable-phyml-mpi --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make clean & make -j${nproc}
+install -Dvm 755 ./src/phyml-mpi${exeext} ${bindir}/phymlMPI${exeext}
+"""
+
+platforms = supported_platforms()
+
+# similar error for below platforms
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/blob/9cee6a3185646db8d27fcc9fded39957e881bff1/F/FastME/build_tarballs.jl#L18-L22

Reply:
I'll work on this and try out the changes you suggested. But my next commit will probably not be until tomorrow